### ISSUE TITLE
User Interface to Allocate Markers

### ISSUE THREAD

**Issue by markpea** [djplaner/moodle-mod_bim#21]
I found the UI to Allocate Markers rather confusing and initially I was uncertain which groups the marker was actually marking. May I suggest that rather than a list display you use a 'checkbox' display here with the list of groups and a box next to each? A 'checked' or ticked box would indicated that the marker was allocated to the associated group. I think that checkboxes would give a clearer display of which groups were allocated to which markers. 
Just my HO.

**Comment by djplaner** [comment]
Good idea.  The list was originally used because I wanted to reduce the amount of screen space consumed, but some other limitations I didn't fix prevented this.  The check boxes are a good idea. Will look at this.
